tests/conftest.py

```diff
@@ -142,3 +143,22 @@ def example_record(db, identity, service, example_data):
     Vocabulary.index.refresh()  # Refresh the index
 
     return record
+
+
+@pytest.fixture(scope='function')
+def lang_data_many(lang_type, lang_data, service, identity):
+    """Create many language vocabulary."""
+    
+    lang_ids = ['fr','tr','gr','ger','es']
+    data = deepcopy(dict(lang_data))
+
+    if 'props' in data.keys():
```

Comment:
If `props` are not needed/supported, I think we should remove from the source fixture